web/src/App.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import LoginScreen from "./components/LoginScreen";
+import TaskApp from "./TaskApp";
+import RealmApolloProvider from "./graphql/RealmApolloProvider";
+import { useRealmApp, RealmAppProvider } from "./RealmApp";
+
+export const APP_ID = "tasktracker-huhcb";
+
+const LoggedIn = ({ children }) => {
+  // Only render children if there is a logged in user.
+  const {app} = useRealmApp();
+  return app.currentUser ? children : null;
+};
+
+const LoggedOut = ({ children }) => {
+  // Only render children if there is not a logged in user.
+  const {app} = useRealmApp();
+  return app.currentUser ? null : children;
+};
+
+export default function App() {
```

Comment:
Could this be simplified and more D.R.Y by removing the LoggedIn() and LoggedOut() methods and putting that logic in the return statement:

```
return(

// ... 

 { app.currentUser? (
        <RealmApolloProvider>
          <TaskApp />
        </RealmApolloProvider>
   ):(<LoginScreen />)
  }

// ...

)
```

Reply:
Yeah I think I YAGNI'd more general react patterns here. Updated to use ternary render instead

Reply:
Hmm, I think you might've forgotten to push your update? I don't see the changes